data/categories.txt

```diff
@@ -2695,7 +2695,7 @@ zh-Hant:纜車
 # Do not add subway_entrance here (pollute a map especially in NY).
 # ToDo: make enhanced subway station highlight with entrances.
 railway-station-subway|@category_transport
-en:3Subway|3tube|3metro|3underground|U+1F687
+en:3Subway Station|3tube|3metro|3underground|U+1F687
```

Comment:
this way both short "subway" and long "metro station" works good.
E.g. before the "metro station" query returned a lot of bus stops with these words in the name, this change shifts priority to actual subway stations.

Reply:
@vng black magic!